.travis.yml

```diff
@@ -47,5 +52,6 @@ deploy:
     file: "netdata*.tar.*"
     on:
       condition: $CC = gcc
+      condition: $TRAVIS_OS_NAME = linux
```

Comment:
According to [this](https://docs.travis-ci.com/user/deployment/) you can only have one condition.